src/SourceBuild/tarball/content/repo-projects/aspnetcore.proj

```diff
@@ -2,14 +2,19 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
+    <OverrideTargetRid>$(TargetRid)</OverrideTargetRid>
+    <OverrideTargetRid Condition="'$(TargetOS)' == 'OSX'">osx-x64</OverrideTargetRid>
```

Comment:
Should this use `$(Platform)` instead of `x64`?  Also would it make sense to use the already defined [PortableRid](https://github.com/dotnet/installer/blob/main/src/SourceBuild/tarball/content/Directory.Build.props#L167) when OSX, FreeBSD, or Windows_NT?  Lastly would it make sense to define the OverrideTargetRid in a common location that be shared by the repo projects that need it?  Not sure if there would be a better name than OverrideTargetRid in that case.

Reply:
Yes, it should use `$(Platform)`. I was planning on splitting that change out into a separate PR, since it causes a different bug